_lifeofinternatpuppet/post_7_and_8.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: page
+title: "Week 7 and Week 8"
+author: Disha-maker
+categories:
+  - IAC Team
+  - Training Sessions
+  - Litmus Demo
+  - QUB Graduate Recruitment and Placement Fair 2020
+tags:
+  - week 7 and 8
```

Comment:
I'm not sure these tags make sense since they'll never be shared with any other posts?

Reply:
I have replied to your email about the tags on the post_7_and_post_8 file which I have been working on. Hope all this information helps you to do a review and merge soon.